examples/panel/chat-with-pdf/requirements.txt

```diff
@@ -1,6 +1,137 @@
-panel
-openai
-chromadb
-pypdf
-langchain
-tiktoken
\ No newline at end of file
+aiohappyeyeballs==2.4.3
+aiohttp==3.9.3
+aiosignal==1.3.1
```

Comment:
All the dependencies can go in a `requirements.lock.txt` file. In this file, you can add the specific package that was needed. You can see the format in the other examples

Reply:
Yes the lock file is for debugging purposes. We can just add the specific package required to resolve the issue `langchain_community.document_loaders` in this file. I'm not sure why all the installed packages need to be mentioned in the `requirements.txt` file

Reply:
It is mentioned in the error: `You can install it using `pip install -U langchain-community`
So just add langchain-community in this file

Reply:
We generally freeze all requirements in the lock file and not the requirements.txt. But maybe @edublancas can suggest?